app/celery/research_mode_tasks.py

```diff
@@ -174,7 +174,7 @@ def ses_notification_callback(reference):
             "messageId": reference,
             "sendingAccountId": "12341234",
             "source": '"TEST" <TEST@notify.works>',
-            "sourceArn": "arn:aws:ses:eu-west-1:12341234:identity/notify.works",
+            "sourceArn": "arn:aws:ses:eu-west-2:12341234:identity/notify.works",
```

Comment:
We probably want to change this from mentioning notify eventually

Reply:
Yeah, as this is a test it shouldn't be too bad.

Reply:
Endpoints and ARNs associated with SES will be corrected when we work on the SES integration ticket.